### ISSUE TITLE
[v4] active:opacity-50 

### ISSUE THREAD

**Issue by exoer** [marklawlor/nativewind#770]
**Describe the bug**
When adding className `active:` the state value is not bound to a handler. 

**To Reproduce**
```
import { View, Text, ScrollView, Pressable, Alert } from 'react-native';
import { useState } from 'react';
import { Stack } from 'expo-router';



import { TextInput } from 'react-native';

export default function Opacity() {
  // const { data: me, isLoading } = useGetMeQuery(undefined);
  const [val, setValue] = useState('');
  console.log();

  const handlePress = () => {
    Alert.alert('Pressed', val);
  };

  return (
    <>
      <Stack.Screen
        options={{
          title: 'Debug Nativewind',
        }}
      />
      <View className="bg-gray-100 gap-4">
        <TextInput
          className="bg-gray-200 p-2"
          value={val}
          onChangeText={setValue}
        />
        {/* this works */}
        <Pressable className="bg-gray-200 p-2" onPress={handlePress}>
          <Text> Press Me</Text>
        </Pressable>
        {/* this does not work. It shows the val as empty */}
        <Pressable
          className="bg-gray-200 p-2 active:opacity-50"
          onPress={handlePress}
        >
          <Text> Press me active:</Text>
        </Pressable>
      </View>
    </>
  );
}
```

**Expected behavior**
Expect that the second button with `className="bg-gray-200 p-2 active:opacity-50"` should show an alert with the value of the entered text. Instead the value is empty.

**Comment by Yohaido159** [comment]
also for me

**Comment by Yohaido159** [comment]
the issue is comming from this line
```
if (state.convertToPressable) {
    // This is an annoying quirk of RN. Pressable will only work if onPress is defined
->    state.props.onPress = state.originalProps.onPress ?? (() => {});
  }
```

when it comment out it will work

**Comment by Yohaido159** [comment]
I think the bug is in this lines
```

  // Merge the styled props with the props passed to the component
  props = {
    ...props,
    ...state.props,
  };

```
you keep the old props and remove the new one
from `packages/react-native-css-interop/src/runtime/native/interop.ts` line 77

**Comment by Yohaido159** [comment]
need to be the opposite 
```
 props = {
    ...state.props,
    ...props,
  };
```

**Comment by exoer** [comment]
> need to be the opposite
> 
> ```
>  props = {
>     ...state.props,
>     ...props,
>   };
> ```

wow @Yohaido159 

I changed `node_modules/react-native-css-interop/dist/runtime/native/interop.js` as you suggest.

This fixed the issue. Now i can have the `active` class on the pressable and it works as expected. (However some of the other styling got messed)

**Comment by Yohaido159** [comment]
@exoer 
could you provide more information about how your style got messed up?
thanks

**Comment by exoer** [comment]
> @exoer could you provide more information about how your style got messed up? thanks

I have temporary abandoned the v4 branch of nativewind and gone back to the v2 branch. Because there were more problems. E.g.`react-native-markdown-display` with an image, there app would error and state that there was an `invalid hook call`

In terms of how the style got messed up when applying your bug-fix. (I wish I had taken a screenshot of the v4). Some elements ended up on separate lines where they before were inline, some spacing was off (possible that I use `gap-x-{n}` or `space-x-{n}` wrong

**Comment by marklawlor** [comment]
Original reported issue about stale event handlers has been fixed in `4.0.24`. Please open new issues for the other issues.
